1. Linearna regresija (poglavlje o regresionoj analizi).ipynb

```diff
@@ -85,7 +85,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
```

Comment:
Curious. I assume all these changes are due to running your own instance of this .ipynb? Do you know maybe why this changes are caught anyway?

Reply:
Maybe you pushed a running instance of this particular .ipynb?

Reply:
Yes, I pulled from the master branch and ran on the local instance. I have opened this particular notebook, but I may have forgotten to close & halt the notebook.